e2e-tests/cypress/integration/project-application-documents.feature

```diff
@@ -186,3 +186,41 @@ Feature: Project Application documents page
         Given I navigate to "Ho Ho Hooo" project Overview page
         When I click on "Project application documents" link
         Then I verify that no documents found text displayed on the page
+
+    Scenario: search returns matching documents
+        Given I navigate to "Hinkley Point C New Nuclear Power Station Material Change 1" project Overview page
+        When I click on "Project application documents" link
+        When I enter text "consultee" into search field
+        And I click on search button
+        Then I can verify that below project documents were returned
+            | Document                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+            | \n            \n              Consultee List sent to the Secretary of State (, 0Bytes)\n            \n            \n              \n                Published by EDF Energy\n              \n            \n            \n              \n                2019-04-16\n              \n              \n                Pre Application\n              \n              \n                Material Change\n              \n            \n                                   |
```

Comment:
I think we can do better than this by stripping out the whitespace/tabs from the content on screen

Reply:
stripped whitespace and line breaks